README.md

```diff
@@ -44,6 +44,8 @@ With GoGovSg, citizens are safe in the knowledge that the links are **official**
 
 ## Getting Started
 
+Make sure you have node version `14`, docker-compose version >= `1.23.1` and Docker version >= `18.09.0` installed.
```

Comment:
have shifted the docker versions up as well to place all the prerequisites together, if it makes sense. happy to take other suggestions as well!